README.md

```diff
@@ -129,14 +129,17 @@ and deploy it.
 fn init $USERNAME/hello
 # build the function
 fn build
-# test it
+# test it - you can pass data into it too by piping it in, eg: `cat hello.payload.json | fn run`
 fn run
-# push it to Docker Hub
-fn push
-# create an app
+# Once it's ready, bump, build and push it to Docker Hub
+fn bump && fn build && fn push
```

Comment:
`fn bump` is actually not required when a new function is being built for the first time.  This section of the quickstart is only covering the case of a new function, not editing an existing function.  The way this is written the first version of the function will be 0.0.2.

`fn bump` is required when the code of an existing function is edited.